Source/Applications/SystemCenter/Controllers/OpenXDA/Meters/OpenXDAMeterController.cs

```diff
@@ -700,5 +700,127 @@ public override IHttpActionResult Patch([FromBody] EventChannel record)
         }
     }
 
+    [CustomView("SELECT " +
```

Comment:
This will pull `EventChannels` and `TrendChannels`

Reply:
@gcsantos-gpa Did you ever add the Flag for TrendChannels? Can we use that here to restrict the view to TrendChannels only?

Reply:
I added a filter to the view to filter out the event channels